packages/orchestration/src/exos/local-chain-facade.js

```diff
@@ -83,6 +98,39 @@ const prepareLocalChainFacadeKit = (
             this.facets.makeAccountWatcher,
           );
         },
+        /**
+         * Get asset info from agoricNames.vbankAsset.
+         *
+         * Caches the query to agoricNames in the first call. Subsequent lookups
+         * are via cache unless a refetch is specified.
+         *
+         * @param {boolean} [refetch] if true, will invalidate the cache
```

Comment:
this option on a getter is surprising to me. I know this fetches but the `get` seems to try to hide that away and it's peeking through here.

It seems rare to me that someone would want to invalidate the whole cache. In this PR there's no use of it, even for tests.

I can see the need for it in a pinch, but consider having an explicit `clearCache` that does the same for all cached info. And of course have tests.

Reply:
> this option on a getter is surprising to me

I guess I'm used to HTTP GET that has no-cache, if-modified-since and all that. I think DNS queries have TTL. Seems like a normal pattern, to me. I thought preserving it from `findBrandInVBank` was pretty straightforward.

I suppose the request for a test is reasonable, though I'm lacking motivation. Maybe I should take it out until I'm sufficiently motivated to test it.

> consider having an explicit `clearCache`

Do you mean rename `refetch`? Or do you mean a separate method? A separate method feels a bit like a TOCTTOU risk, though as long as it's synchronous, it doesn't actually have that problem. It just feels like odd protocol design.

> It seems rare to me that someone would want to invalidate the whole cache

the cache is one object. it's not a store with parts any more.

Reply:
>> It seems rare to me that someone would want to invalidate the whole cache

> the cache is one object. it's not a store with parts any more.

The current fetching pattern is to grab all vbank entries at once, so invalidating the entire cache makes sense to me. 

The original motivation of `refetch` for `findBrandInVBank` was to give an escape hatch in case a vbankAsset entry changes. It's weird to cache something and not give the consumer an option to clear it. If we can make a case that vbankAsset is append-only, I think it's ok to leave out. However, I don't think we can - at some point it was proposed to add add'l keys to VbankAsset like baseDenom and channel/trace.

For `getVBankAssetInfo` it seems more important to include since we're caching a list we expect to get new entries over time.

Reply:
>I guess I'm used to HTTP GET that has no-cache

Yeah, because that's a network request. In JS a `getFoo` is generally a synchronous getter. In JS when you want an HTTP GET you call `fetch`.

>I suppose the request for a test is reasonable, though I'm lacking motivation. Maybe I should take it out until I'm sufficiently motivated to test it.

SGTM. Either test or remove.

> A separate method

Yeah, for the reasons Patrick articulated. It would be synchronous.

Reply:
>  because that's a network request

this is a request to a remote service too, no?

I don't see Patrick giving any reasons for a separate method.

Reply:
leaving it out for now